README.md

```diff
@@ -55,6 +63,9 @@ CentOS Linux 8. It does not support CentOS Stream.
    to Oracle Linux. Users may need to enable additional repositories to obtain
    updates for packages already installed (see [issue #1](https://github.com/oracle/centos2ol/issues/1)).
 
+## Debugging
+Run `sudo bash -x centos2ol.sh` to switch your CentOS instance to Oracle Linux in a debug mode. This will print a trace of commands and their arguments or associated word lists after they are expanded but before they are executed.
```

Comment:
```suggestion
Run `sudo bash -x centos2ol.sh` to switch your CentOS instance to Oracle Linux in debug mode. This will print a trace of commands and their arguments or associated word lists after they are expanded but before they are executed.
```